Composer/packages/electron-server/scripts/copy-artifacts.js

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+const path = require('path');
+
+const fs = require('fs-extra');
+
+const electronBuildConfig = require('../electron-builder-config.json');
+
+const { log } = require('./common');
+
+const oneauthSource = () => {
+  const oneauthPath = path.resolve(__dirname, '../oneauth-temp');
+  if (['win32', 'darwin'].includes(process.platform) && fs.existsSync(oneauthPath)) {
+    return { source: oneauthPath, dest: 'oneauth', force: true };
+  } else {
+    log.info('Skipping OneAuth. Either on an unsupported platform or it has not been installed to %s.', oneauthPath);
+  }
+};
+
+const sources = [
+  // extensions
+  { source: path.resolve(__dirname, '../../../../extensions'), dest: 'extensions' },
+  // runtimes
+  { source: path.resolve(__dirname, '../../../../runtime'), dest: 'runtime' },
+  // form-dialog templates
+  {
+    source: path.resolve(__dirname, '../../../node_modules/@microsoft/bf-generate-library/templates'),
+    dest: 'form-dialog-templates',
+  },
+  // oneauth
+  oneauthSource(),
+].filter(Boolean);
+
+let destinationDir;
+switch (process.platform) {
+  case 'darwin': {
+    const productName = electronBuildConfig.productName;
+    destinationDir = path.resolve(__dirname, `../dist/mac/${productName}.app/Contents/Resources/app.asar.unpacked`);
+    log.info('Mac detected. Copying artifacts to: ', destinationDir);
+    break;
+  }
+
+  case 'linux':
+    destinationDir = path.resolve(__dirname, '../dist/linux-unpacked/resources/app.asar.unpacked');
+    log.info('Linux detected. Copying artifacts to: ', destinationDir);
+    break;
+
+  case 'win32':
```

Comment:
is `win64` a valid case?

Reply:
win64 is not a possible value of `process.platform`:

https://nodejs.org/api/process.html#process_process_platform